scripts/test-runner-filter.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+try {
+  const minimatch = require('minimatch');
+  const childProcess = require('child_process');
+  const gitDiffRange = (
+    process.env.TRAVIS_COMMIT_RANGE || 'master..HEAD'
+  ).replace('...', '..');
+  const changedFiles = childProcess
+    .execSync(`git diff --name-only ${gitDiffRange}`, {encoding: 'utf8'})
+    .split('\n')
+    .slice(0, -1);
+
+  const testRelatedFilePatterns = [
+    'scripts/test-runner-filter.js',
+    'bin/**/*',
+    'lib/**/*',
+    'test/**/*',
+    '.mocharc.yml',
+    '.wallaby.js',
+    'index.js',
+    'package.json',
+    'package-lock.json',
+    'package-scripts.json',
+    '.nycrc',
+    '.travis.yml',
+    'browser-entry.js',
+    'karma.conf.js',
+    'mocha.*'
+  ];
```

Comment:
Should these be ordered by likelihood of being a hit, then maybe lexically?
Would think `'lib/**/*'`, `'test/**/*'`, `'bin/**/*'` should be first three (in that order). Alphabetize the rest.